model/workflow.go

```diff
@@ -501,36 +501,31 @@ type ContinueAs struct {
 	WorkflowExecTimeout WorkflowExecTimeout `json:"workflowExecTimeout,omitempty"`
 }
 
+//type testFOrUnmarshall ContinueAs
+
 func (c *ContinueAs) UnmarshalJSON(data []byte) error {
-	rewf := make(map[string]interface{})
-	if err := json.Unmarshal(data, &rewf); err != nil {
+	continueAs := make(map[string]json.RawMessage)
```

Comment:
@lsytj0413 or something like this.

Reply:
I don't know what the different of previous implement，it always unmarshal multi times right？
There is some explain how StateExecTimeout style works https://github.com/serverlessworkflow/sdk-go/pull/76#issuecomment-1253100748

Reply:
did a small update, please take a look.